runtime/src/nonce_keyed_account.rs

```diff
@@ -1,314 +1,292 @@
 use {
     solana_program_runtime::{ic_msg, invoke_context::InvokeContext},
     solana_sdk::{
-        account::{ReadableAccount, WritableAccount},
-        account_utils::State as AccountUtilsState,
         feature_set::{self, nonce_must_be_writable},
         instruction::{checked_add, InstructionError},
-        keyed_account::KeyedAccount,
         nonce::{self, state::Versions, State},
         pubkey::Pubkey,
         system_instruction::{nonce_to_instruction_error, NonceError},
         sysvar::rent::Rent,
+        transaction_context::InstructionContext,
     },
     std::collections::HashSet,
 };
 
-pub trait NonceKeyedAccount {
```

Comment:
@jackcmay I think you meant removing `NonceKeyedAccount` instead of the `ReadableAccount`, `WritableAccount` Traits, right?